sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1820,10 +1826,136 @@ void testDetermineCharsetFromMessageFromHttpHeader(@TempDir final File dir) thro
         }
     }
 
+    /**
+     * Ensures that the test run is not invalidated when an encoding error is detected
+     * but SummarizeMessageEncodingErrors is set to true in the configuration.
+     * Also checks that the encoding error is nevertheless counted correctly for the summary.
+     *
+     * @param dir - a temporal directory.
```

Comment:
```suggestion
     * @param dir - a temporary directory.
```
and all other occurences.